src/components/feed/AddNode/AddNode.tsx

```diff
@@ -340,12 +354,18 @@ const AddNode: React.FC<AddNodeProps> = ({
 
   return (
     <React.Fragment>
+      {isCreator ? (
       <Button icon={<MdOutlineAddCircle />} type="button" onClick={toggleOpen}>
         Add a Child Node{" "}
         <span style={{ padding: "2px", color: "#F5F5DC", fontSize: "11px" }}>
           ( C )
         </span>
       </Button>
+      ) : (
+      <button disabled={true} >
```

Comment:
When a button is disabled, it is good for there to be a popup/tooltip explaining why.

Reply:
Okay

Reply:
Shall I use jQuery UI library?

Reply:
Would the message "You are not the feed owner" be okay?